projects/sdr_paper/pytorch_experiments/experiments_paper.cfg

```diff
@@ -21,6 +21,9 @@ create_plots = False
 count_nonzeros = True
 test_noise_every_epoch = True
 
+use_batch_norm = True
```

Comment:
should we default to False for MNIST since batch norm didn't help?

Reply:
yes